migration/src/main/resources/META-INF/rewrite/ToLatestSingleConstraintAssertion.yml

```diff
@@ -0,0 +1,17 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: ai.timefold.solver.migration.ToLatestSingleConstraintAssertion
+displayName: 'Use non-deprecated SingleConstraintAssertion methods'
+description: 'Use `penalizesBy/rewardsWith(String, int)` instead of `penalizesBy/rewardsWith(int, String)` on `SingleConstraintAssertion` tests.'
```

Comment:
I'd prefer we have a Java class for this. See how I created the `NullableRecipe`.
The main reason is that if we did every such thing as a top-level YML recipe, we would soon have too many of them. I'd like to keep that folder relatively clean.